source/index.html

```diff
@@ -1,12 +1,153 @@
+              <a class="nav-list__link" href="photo.html">
+                Фото и видео
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="form.html">
+                Форма отзыва
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="https://htmlacademy.ru/">
+                HTML Academy
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="page-main">
+      <div class="wrapper">
+        <h1 class="visually-hidden">Город Sedona - небольшой городок в Аризоне</h1>
+        <section class="slogan">
+          <div class="slogan__wrapper">
+            <!-- Картинка гор будет добавлена как bacjground -->
+            <img src="../source/img/text-sedona-index.svg" alt="Добро пожаловать в прекрасную Sedona. Потому что Гранд Каньон отстой">
+          </div>
+        </section>
+        <section class="content">
+          <header class="content__header">
+            <h2 class="visually-hidden">Причины, почему Sedona лучше Гранд Каньона</h2>
+            <p class="content__slogan">СЕДОНА — небольшой  городок в Аризоне,  заслуживающий большего!</p>
+            <p class="content__intro">Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
+          </header>
+          <div class="content__card content__card--blue">
+            <h3 class="content__name">Настоящий городок</h3>
+            <span class="content__number">- №1 -</span>
+            <p class="content__description">Седона не аттракцион для туристов, там течет своя жизнь</p>
+          </div>
+          <img 
+            class="content__image content__image-1" 
+            src="../source/img/main-city-desktop@1x.jpg"
+            srcset="
+              ../source/img/main-city-mobile@1x.jpg 320w,
+              ../source/img/main-city-tablet@1x.jpg 768w,
+              ../source/img/main-city-desktop@1x.jpg 1024w
+              "
+            sizes="(max-width: 320px) 280px, (max-width: 768px) 700px, 1024px" 
+            alt="фасад здания">
+          <div class="content__card-wrapper">
+            <div class="content__card content__card--house">
+              <h3 class="content__name">Жилье</h3>
+              <p class="content__desctiption">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
+            </div>
+            <div class="content__card content__card--food">
+              <h3 class="content__name">Еда</h3>
+              <p class="content__desctiption">Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+            </div>
+            <div class="content__card content__card--gifts">
+              <h3 class="content__name">Сувениры</h3>
+              <p class="content__desctiption">Не только китайского, но и местного производства!</p>
+            </div>
+          </div>
+          <img 
```

Comment:
Seems you'll need a wrapper for this image and other texts of this block

Reply:
added some wrappers. Need to discuss